src/main/kotlin/com/buildit/bookit/v1/location/bookable/BookableController.kt

```diff
@@ -71,14 +75,11 @@ class BookableController(private val bookableRepository: BookableRepository, pri
             end.atStartOfDay(timeZone).toInstant()
         )
 
-        val allBookings = bookingRepository.getAllBookings()
-        return bookableRepository.getAllBookables()
-            .filter { it.locationId == locationId }
+        return bookableRepository.findByLocation(location)
             .map { bookable ->
                 val bookings = when {
                     "bookings" in expand ?: emptyList() ->
-                        allBookings
-                            .filter { booking -> booking.bookableId == bookable.id }
+                        bookingRepository.findByBookable(bookable)
```

Comment:
Now we have an O(n^2) (or some variety of such) thing here, I think, WRT database calls.  With the data we have today, no big deal, but someday ...

Reply:
yess..... thanks for point that out... forgot to clean this up... will leave in for this pr and fix right after

Reply:
although, maaaaaybe some hibernate caching (all in the same transaction) alleviates us from being really dumb about it

Reply:
I can't complain.  Really nice work here.  I'm loving the reduction in code.  Spring Data is pretty darn nice.